<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraFocalLengthDialog</class>
 <widget class="QDialog" name="CameraFocalLengthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border-radius: 20px;
	background: transparent;
	border: 1px solid rgba(255, 255, 255, 51);
}
QWidget {
	font-family: &quot;思源黑体&quot;;
	color: white;
	background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#centralwidget{
	border-radius: 20px;
	background: rgba(61, 71, 112, 1);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>48</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_title" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget #widget_title{
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(78, 228, 224, 0),
        stop:0.1919 rgba(77, 153, 245, 64),
        stop:1 rgba(78, 218, 227, 0));
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>14</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_title_icon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget {
    background: transparent;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
           <property name="pixmap">
            <pixmap resource="../ui.qrc">:/assets/png/tooltip.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_title_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>思源黑体</family>
             <pointsize>21</pointsize>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget {
    background: transparent;
}</string>
           </property>
           <property name="text">
            <string>相机焦距测量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>X</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/client/resources.qrc">
             <normaloff>:/assets/button/close.png</normaloff>:/assets/button/close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <family>思源黑体</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>相机焦距测距信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>等分机位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_parts">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{
    background: rgba(37, 46, 74, 1);
    color: rgba(129, 141, 179, 1);
    border: 1px solid rgba(80, 95, 138, 1);
}


/* 下拉列表的样式 */
QComboBox QAbstractItemView {
    border: 1px solid rgba(80, 95, 138, 1);
    background-color: rgba(37, 46, 74, 1);  /* 下拉背景色 */
    selection-background-color: rgba(80, 95, 138, 1);  /* 选中项背景色 */
    color: rgba(129, 141, 179, 1);  /* 文字颜色 */
}</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_CameraFocal">
        <property name="font">
         <font>
          <family>思源黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section {
    background: rgba(73, 126, 229, 1);
	padding: 8px;
	border: 1px solid #3e8e41;
	font-size: 14px;
	font-weight: bold;
}
QTableWidget {
	background: rgba(61, 71, 112, 1);
	gridline-color: #d4d4d4;
	selection-background-color: #a6a6a6;
	border: 2px solid grey;
	border-radius: 5px;
}
QTableWidget::item {
	padding: 8px;
}
QTableWidget::item:selected {
	background-color: rgb(0, 85, 127);

}</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::PenStyle::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>相机轨面高(m)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>扫描仪轨面高(m)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>相机焦距值列表(m)</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>test</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>1.8</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0.556</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>{0.898}</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0.556</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>0.556</string>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string>0.556</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_action">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_action">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_start">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
            </property>
            <property name="text">
             <string>开始扫描</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_get">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
            </property>
            <property name="text">
             <string>计算焦距</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_new_info">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
            </property>
            <property name="text">
             <string>新建数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_goto_reset_page">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
            </property>
            <property name="text">
             <string>修改数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_reset">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="text">
             <string>选中的数据项:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_reset">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 10px;
background: rgba(37, 46, 74, 1);
border: 1px solid rgba(80, 95, 138, 1);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reset">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
            </property>
            <property name="text">
             <string>确认修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reset_page_quit">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>思源黑体</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(250, 77, 100, 1),
        stop:1 rgba(240, 112, 79, 1));</string>
            </property>
            <property name="text">
             <string>退出修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../main/client/resources.qrc"/>
  <include location="../ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraFocalLengthDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
